frontend/packages/dev-console/src/components/deployments/ContainerField.tsx

```diff
@@ -15,10 +15,10 @@ const ContainerField: React.FC = () => {
   const containerName = containers[0]?.name;
   return (
     <FormGroup fieldId="container-name">
-      <span style={{ fontWeight: 'bold' }}>
+      <span style={{ fontWeight: 'bold', position: 'relative' }}>
         <Trans t={t} ns="devconsole">
           Container: <ResourceIcon kind={ContainerModel.kind} />
-          {{ containerName }}
+          {containerName}
```

Comment:
I think the old version was correct to use a i18n variable here that will not be translated.

The old version looks like this with pseudo localization:

![image](https://user-images.githubusercontent.com/139310/219011662-caa22fec-28f6-4818-8f8e-d336d4d41398.png)

But now the string is added and then translated:

![image](https://user-images.githubusercontent.com/139310/219011772-52b32244-5526-49f4-8619-8f0f8c31bbc6.png)

With that the translation doesn't work because it doesn't match:

![image](https://user-images.githubusercontent.com/139310/219011969-92092293-108f-4de0-9635-28990100965b.png)

```suggestion
          {{ containerName }}
```